Gemfile

```diff
@@ -57,6 +57,7 @@ end
 group :development, :test, :testing do
   gem 'factory_girl_rails', '~> 4.5.0'
   gem 'faker', '~> 1.4.3'
+  gem 'pry'
```

Comment:
Probably shouldn't include the same gem twice.  Bundler will give you a warning.  :)